tests/acceptance/fixtures/test_libdoc_file.xml

```diff
@@ -0,0 +1,561 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Can you reduce this file to minimal size required for testing purposes?

Reply:
Changed to Easter Builtin Library, which results in 15 lines